### PR TITLE
Refactor/workshop

### PR DESCRIPTION
**Pull request by BrokenShell**
# Refactor Workshop
Fixes several bugs and the code follows best practice a bit better (PEP8).

## Contributors
- jonathan mcgraw
- Yefim Gorodnitskiy
- Brogan Stich
- Eisenhower Lancheros
- John Baker
- Scott Reynders
- Nic Papenburg
- Elisa Mala

## PR Type
- [x] Bug fix

## Checklist:
- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by jinjahninjah — CHANGES_REQUESTED** [review]
:shipit:

**Review by jinjahninjah — APPROVED** [review]

**Review by yefim-g — APPROVED** [review]
Definitely a beneficial workshop!

File: app/api.py [hunk 1]
```diff
@@ -58,31 +61,35 @@ async def create(collection: str, data: Dict):
...
+          "industry_knowledge": false,
+          "pair_programming": false,
+          "your_hope": "String",
+          "need": "Laptop",
+          "parole_restriction": true,
+          "disability": true,
+          "work_status": true,
+          "assistance": false,
+          "other_info": "Notes"
+    }
```
Comment (BakerJr1904) [inline app/api.py:87]: I wish the f string worked; that would have been nice, to automate changes in the database with the docstring.

**Review by BakerJr1904 — APPROVED** [review]
It was a great time doing this PR

**Review by Zack-Quintana — APPROVED** [review]
Doesn't seem to be any changes needing to be made. Nice job

**Review by hollberg — APPROVED** [review]
Adding a 'neutral' option makes this feature much more useful.

**Pull request merged**